packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -71,10 +72,12 @@ function DatabaseList(props: State) {
 function ConnectButton({
   documentUri,
   dbUri,
+  protocol,
   onConnect,
 }: {
   documentUri: string;
   dbUri: string;
+  protocol: string;
```

Comment:
Could you use `GatewayProtocol` as the type here? https://github.com/gravitational/webapps/blob/79547b2d7399bfa611e3fd1e9dd04452a718ad2b/packages/teleterm/src/services/tshd/types.ts#L13-L23

Reply:
Done